src/mapeo-manager.js

```diff
@@ -145,4 +160,58 @@ export class MapeoManager {
 
     return project
   }
+
+  /**
+   * @returns {Promise<Array<ProjectInfo & Pick<ProjectValue, 'name'>>>}
+   */
+  async listProjects() {
```

Comment:
All the shenanigans about having a shared index writer were so that we did not have to create project instances to get the project name, and instead we can do:

```js
this.#db.select().from(projectTable).all()
```

Reply:
addressed via https://github.com/digidem/mapeo-core-next/pull/208/commits/b29fa8bc99ff3e99eee41374a19b41842c5874e4